engineering/roles/engineering-manager.md

```diff
@@ -0,0 +1,88 @@
+
+### You can keep track of, prioritize, and manage multiple projects
+
+We're a growing team, and there's no shortage of things you could be doing in a day. You'll carve out time for functional projects and ensure the engineers on your teams solve issues that matter -- making our cloud infrastructure more efficient and agile, building a culture of reliability, and delivering outstanding service to the communities that we serve.
+
+### You have excellent communication skills
+
+You understand and appreciate what your stakeholders need and you can speak their language. You’ll regularly work with our Product, Community Guidance, and Partnerships teams, balancing engineering concerns (such as technical debt) with product and service concerns. Not only do you know how to share your knowledge with the team and write asynchronously-consumable documentation, but you know how to communicate your vision effectively with software and support teams.
+
+### You have a background in Site Reliability Engineering
+
+You’re comfortable talking about SLOs, incident management, and building a culture of reliability. You have empathy for the communities we serve and are eager to make improvements for them. You seek to reduce toil and understand what it means to take a software engineering approach to operations.
+
+### You know the cloud
+
+You’ve designed and maintained highly available, cloud-based infrastructures in AWS or another cloud offering. You understand how to leverage infrastructure-as-code tools and have experience implementing best practices for reliability and observability.  We currently use Terraform, Kubernetes, GitHub, and JupyterHub.
+
+### You have experience with open source communities
+
+You’ve worked in one or more multi-stakeholder community-driven open source projects. You understand the challenges and messiness that often comes with open communities, you have experience contributing to them, and an interest in growing a team that does most of its work via upstream contributions. While 2i2c does not directly manage any open source communities, it aims to be an active participant in many, and upstreaming improvements to open source projects is a core part of our workflow.
+
+### You value our values
+
+2i2c is a mission-driven organization defined by [our values and principles](https://2i2c.org/about/). You see how these values can empower meaningful work, thrive in a collaborative setting, and are eager to continue growing excited to be part of the team.
+
+## Salary rationale
+
+We are currently discussing our salary bands and levels [as per this proposal](https://github.com/2i2c-org/meta/issues/402#issuecomment-1419224044). According to that proposal (most likely to be implemented and already a reference for recent hires), the base salary associated with a Engineer Manager role (level 4) is $**151,200**.
```

Comment:
```suggestion
We are currently discussing our salary bands and levels for Engineering [as per this proposal](https://github.com/2i2c-org/meta/issues/402#issuecomment-1419224044). According to that proposal (most likely to be implemented and already a reference for recent hires), this role is currently associated with the base salary corresponding to an Engineer Manager (level 4) role: https://docs.google.com/spreadsheets/d/1D_-RzXhyWw8jFs7KQQXKVRV4VrmOyXjQni2UTnk1044/edit#gid=0.
```
I think it is important to ack the dynamic nature of the salary and this is why I believe we should link the model we have using instead of having a static value. We do the same for other roles described on the team compass.